hack/download-image.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+# Copyright 2022 Chaos Mesh Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e
+
+function usage() {
+    cat <<'EOF'
+This command downloads and imports the chaos-mesh image from the github artifacts
+
+Usage: hack/download-image.sh
+
+    -h show this message and exit
+    -r the github repository which running the
```

Comment:
"-r the github repository which running the " looks a little weird?

Reply:
Fixed.